### PR TITLE
Economic Graphs Output

### LINKED ISSUE #31: Economic Graph Output

### PR DESCRIPTION
**Pull request by aj3x**
Add economics chart to finances screen

* fix economics calculations
* fix chart offset
* default values to 0 for chart

https://user-images.githubusercontent.com/15078358/115453810-4573e680-a1dd-11eb-8062-3cce64824f3f.mov

Closes #31

### CODE REVIEW

File: src/services/ChartService.js [hunk 1]
```diff
@@ -243,9 +243,75 @@ const getColor = (index, isHover = false) => {
   return isHover ? hoverColors[index-1] : colors[index-1];
 };
 
+function FinancingSavingsBar (props){
+  // const monthlyPayments = 386.27;
+  // const monthlySavings = 458.81;
```
Comment (scottborys) [inline src/services/ChartService.js:248]: What are these commented numbers?

**Review by scottborys — APPROVED** [review]

File: src/services/ChartService.js (hunk 1 above)
Comment (aj3x) [reply src/services/ChartService.js:248]: The values that were in the excel sheet for testing

Comment (scottborys) [reply src/services/ChartService.js:248]: Should this be indicated somewhere? Or these values removed?

Comment (brightbuildings) [reply src/services/ChartService.js:248]: Hi, the monthlypayments and monthlysavings are useful indications of affordability.

**Pull request merged**